src/main/java/envoy/event/GroupResizeEvent.java

```diff
@@ -53,45 +53,44 @@
 	 * Initializes a {@link GroupResizeEvent} through a Contact where the name has
 	 * already been set.
 	 *
-	 * @param user          the {@link User} who wants to join or leave a group
-	 * @param group         the {@link Group} he wants to join or leave
-	 * @param operationType whether the user should be removed from this group. If
-	 *                      false,
-	 *                      this user will be added
+	 * @param user      the {@link User} who wants to join or leave a group
+	 * @param group     the {@link Group} he wants to join or leave
+	 * @param operation whether the user should be removed from this group. If
+	 *                  false, this user will be added.
 	 * @since Envoy Common v0.2-alpha
 	 */
-	public GroupResizeEvent(User user, Group group, OperationType operationType) {
+	public GroupResizeEvent(User user, Group group, Operation operation) {
 		super(user.getID());
 		if (group.getMemberIDs().contains(user.getID())) {
-			if (operationType.equals(OperationType.ADD)) throw new IllegalStateException(
+			if (operation.equals(Operation.ADD)) throw new IllegalStateException(
 					"Cannot add " + user + " to group " + group.getID() + " because he is already a member of this group");
-		} else if (operationType.equals(OperationType.REMOVE))
+		} else if (operation.equals(Operation.REMOVE))
 			throw new IllegalStateException("Cannot remove " + user + " from group " + group.getID() + " because he is no part of this group");
 		groupID				= group.getID();
-		this.operationType	= operationType;
+		this.operation	= operation;
 	}
 
 	/**
 	 * @return the ID of the {@link Contact} this event is related to
 	 * @since Envoy Common v0.2-alpha
 	 */
-	public long getGroupId() { return groupID; }
+	public long getGroupID() { return groupID; }
 
 	/**
 	 * @return the operationType
 	 * @since Envoy Common v0.1-beta
 	 */
-	public OperationType getOperationType() { return operationType; }
+	public Operation getOperationType() { return operation; }
 
 	/**
-	 * @param operationType the operationType to set
+	 * @param operation the operationType to set
 	 * @since Envoy Common v0.1-beta
 	 */
-	public void setOperationType(OperationType operationType) { this.operationType = operationType; }
+	public void setOperationType(Operation operation) { this.operation = operation; }
 
 	/**
 	 * {@inheritDoc}
 	 */
 	@Override
-	public String toString() { return String.format("GroupResizeEvent[userid=%d,groupid=%d,operationType=%b]", get(), groupID, operationType); }
+	public String toString() { return String.format("GroupResizeEvent[userid=%d,groupid=%d,operation=%b]", get(), groupID, operation); }
```

Comment:
The `operation` is not a `boolean` anymore, so the format string needs to be adjusted.
```suggestion
	public String toString() { return String.format("GroupResizeEvent[userid=%d,groupid=%d,operation=%s]", get(), groupID, operation); }
```